src/ansys/fluent/core/solver/error_message.py

```diff
@@ -25,3 +25,55 @@ def allowed_values_error(
     context: str, trial_name: str, allowed_values: List[str]
 ) -> ValueError:
     return ValueError(allowed_name_error_message(context, trial_name, allowed_values))
+
+
+class FluentConnectionError(ValueError):
+    """Custom Fluent connection errors."""
+
+    class PortNotProvidedError(ValueError):
+        def __init__(self, error):
+            super().__init__(error)
+
+    class StartTimeoutError(RuntimeError):
+        def __init__(self, error):
+            super().__init__(error)
+
+    class RemoteNotSupportedError(ValueError):
+        def __init__(self, error):
+            super().__init__(error)
+
+    class WaitTypeError(TypeError):
+        def __init__(self, error):
+            super().__init__(error)
+
+
+class LauncherError(RuntimeError):
```

Comment:
Why does this inherit from `RuntimeError`?

Reply:
It's removed now. Thank you.